spec/features/projects/project_custom_fields/overview_page/dialog/validation_spec.rb

```diff
@@ -290,6 +290,12 @@
           let(:custom_field) { integer_project_custom_field }
```

Comment:
This can be inline in shared example, as it is same in both usages and has a bit confusing name, as it is the edited field, but not the calculated value CF, that I would expect to be assigned to variable custom_field in this describe block